components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,160 @@
+
+  return (
+    <Container id={id} mt={marginTop} mb={marginBottom} className={styles.root} {...props}>
+      {heading && <Heading size={3} as={headingLevel} className={styles.heading}>{heading}</Heading>}
+      {intro && <div className={styles.intro}>{intro}</div>}
+      {showControls && (
+        <FlexBox justifyContent="end" className={styles.controls}>
+          <CtaButton
+            disabled={allExpanded}
+            variant="ghost"
+            size="small"
+            color={isDarkTheme ? 'white' : 'black'}
+            icon="plus"
+            iconProps={{ className: styles.expandAllIcon}}
+            onClick={expandAll}
+            className={styles.controlButton(isDarkTheme)}
+          >
+            Expand All
+          </CtaButton>
+          <CtaButton
+            disabled={allCollapsed}
+            variant="ghost"
+            size="small"
+            color={isDarkTheme ? 'white' : 'black'}
+            icon="minus"
+            iconProps={{ className: styles.collapseAllIcon}}
+            onClick={collapseAll}
+            className={styles.controlButton(isDarkTheme)}
+          >
+            Collapse All
+          </CtaButton>
+        </FlexBox>
+      )}
+      <ul className={styles.list}>
+        {items?.map((item, index) => (
+          <li key={item._uid} className={styles.listItem}>
+            <Heading as={itemHeadingLevel} color={isDarkTheme ? 'white' : 'black'} leading="tight" className={styles.itemHeading}>
+              <CtaButton
+                id={`button-${item._uid}`}
+                ref={index === 0 ? firstItemRef : undefined}
+                onClick={() => toggleItem(index)}
+                variant="unset"
+                color={isDarkTheme ? 'white' : 'black'}
+                aria-expanded={openItems[index] || false}
+                aria-controls={`content-${item._uid}`}
+                className={styles.button}
+              >
+                <span aria-hidden="true" className={styles.bar} />
+                {item.heading}
+                <HeroIcon icon={openItems[index] ? 'minus' : 'plus'} className={styles.circleIcon} />
+              </CtaButton>
+            </Heading>
+            <m.div
+              role="region"
+              aria-labelledby={`button-${item._uid}`}
+              id={`content-${item._uid}`}
+              aria-hidden={!openItems[index]}
+              animate={{
+                height: openItems[index] ? 'auto' : 0,
+                visibility: openItems[index] ? 'visible' : 'hidden',
+              }}
+              initial={false}
+              transition={{
+                height: { duration: 0.2, ease: 'easeIn' },
+                /**
+                 * Use a shorter duration for visibility to prevent links in content being focusable
+                 * while the content is animating out while collapsing
+                 */
+                visibility: { duration: 0.1 },
+              }}
```

Comment:
@sherakama I suspected that the issue you experienced was due to the original 0.3 sec duration when the visibility animates from hidden to visible (visibility hidden is what makes the content un-focusable). I have lower the duration to 0.1 sec. Previously I occasionally was able to reproduce what you experienced, but now it seems to be ok for me. Please retest and let me know 😄

Reply:
Found a better way to deal with it
https://github.com/SU-SWS/ood-giving-campaign/pull/402#discussion_r2004401640